Paraswap-Agents/Role-Changes/src/utils.ts

```diff
@@ -0,0 +1,54 @@
+import { LogDescription, Finding, FindingSeverity, FindingType } from "forta-agent";
+
+export const SWAPPER_CONTRACT = "0xDEF171Fe48CF0115B1d80b88dc8eAB59176FEe57"; //AugustusSwapper contract
+export const EVENTS_ABI = [
+  "event RoleAdminChanged(bytes32 indexed role, bytes32 indexed previousAdminRole, bytes32 indexed newAdminRole)",
+  "event RoleGranted(bytes32 indexed role, address indexed account, address indexed sender)",
+  "event RoleRevoked(bytes32 indexed role, address indexed account, address indexed sender)",
+];
+
+export const createFinding = (log: LogDescription) => {
+  if (log.name == "RoleAdminChanged") {
+    return Finding.fromObject({
+      name: "Admin role change detected on AccessControl contract",
```

Comment:
I recommend using a `switch` statement instead of an `if/else` chain here.
If you would like an example, see [here](https://github.com/NethermindEth/Forta-Agents/blob/%40paraswap/delegateCalls/Paraswap-Agents/delegate-calls/src/finding.ts)

Also, for the agents you build from now on I'd recommend trying to make one `createFinding` function that works for both your `agent.ts` and `agent.spec.ts`. 
It's going to be too much of an effort to make these changes to this existing agent, but when you're working on your next one keep it in mind.
The code for `createFinding` in the agent and tests are always very similar so it makes sense to try and have one function that works for both where possible.

Reply:
Not too much in line with this recommendation. 

Reasons:
- That function is going to be shared between the logic and the tests that can introduce errors. It is not something high probable to happen here because it is a pretty simple function and minimal sharing like this can be allowed, but in general sharing is not a good practice while testing.
- It is a lot easier to have different functions with custom parameters in testing an even many functions instead of only one `createFinding`, in order to achieve simplicity and clarity. For example I never used a `LogDescription` object in tests, so using the same function will add an extra work in the test to create that object

Reply:
Also not worthy for a `request changes` comment because the approach used before works correctly, you are just suggesting a different approach with only syntax's changes